db/migrate/20210715145948_create_subjects.rb

```diff
@@ -1,7 +1,7 @@
 class CreateSubjects < ActiveRecord::Migration[6.1]
   def change
     create_table :subjects do |t|
-      t.string :name
```

Comment:
At the risk of telling you something you already know, normally you cannot edit migrations. In this case it's okay since you don't have the app in production. Let's talk more about it!